### PR TITLE
Added event for London-GDS

### PR DESCRIPTION
**Pull request by georgeracu**
This year GDS is hosting a GDCR on 17 of November.

### CODE REVIEW

**Comment by rradczewski** [comment]
Hi @georgeracu , looks like Travis rejected that job ("Abuse detected", whatever that means) and the EventBrite link doesn't work. Could you fix the link? Maybe Travis will work again by then :+1

File: _data/events_gdcr2018/London-GDS.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "title": "GDCR18 @GDS London",
+  "url": "https://www.eventbrite.com/e/gdcr18-gds-london-tickets-52153436332",
```
Comment (rradczewski) [inline _data/events_gdcr2018/London-GDS.json]: Doesn't exist for me

Comment (georgeracu) [reply _data/events_gdcr2018/London-GDS.json]: Yes, the link to the event is not public yet. Should I get back to raise it again once the link goes public?
Thanks :)

Comment (rradczewski) [reply _data/events_gdcr2018/London-GDS.json]: Yeah let's wait until people can actually sign up :+1:

Comment (georgeracu) [reply _data/events_gdcr2018/London-GDS.json]: Eventbrite link has been updated :)

**Comment by heathd** [comment]
Hi @rradczewski. We've now made the eventbrite link live. Could you take another look when you get a chance? Thanks!

**Comment by thodorisbais** [comment]
Welcome @georgeracu , @heathd , have a good one 🎉

**Pull request merged**